shinobi.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "shinobi",
+  "plugin_schema": "2",
+  "release": "11.3-RELEASE",
+  "artifact": "https://github.com/allensandiego/iocage-plugin-shinobi.git",
+  "pkgs": [
+	"git",
+	"ffmpeg",
+	"x264",
+	"x265",
+	"node",
+	"npm",
+	"mysql57-server",
+	"mysql57-client"
+  ],
+  "packagesite": "http://pkg.FreeBSD.org/${ABI}/latest",
+  "fingerprints": {
+		  "plugin-default": [
+				  {
+				  "function": "sha256",
+				  "fingerprint": "b0170035af3acc5f3f3ae1859dc717101b4e6c1d0a794ad554928ca0cbb2f438"
+			  }
+			]
+	},
+	"official": false
```

Comment:
Can we fix the indentation here please ? ( and above as well )

Reply:
Made some fixes on the indent. Used vs-code auto format. Let me know if it's still not ok.

Reply:
@allensandiego please fix the indentation to use 4 spaces as indent. Thanks!